meta-resin-common/recipes-devtools/kernel-modules-headers/kernel-modules-headers.bb

```diff
@@ -22,7 +22,7 @@ do_configure[noexec] = "1"
 
 do_compile() {
     mkdir -p kernel_modules_headers
-    if [ "${TRANSLATED_TARGET_ARCH}" = "x86-64" ] || [ "${TRANSLATED_TARGET_ARCH}" = "i686" ]; then
+    if [[ "${TRANSLATED_TARGET_ARCH}" =~ ^.+86 ]]; then
```

Comment:
Hm I wonder about an arch that randomly has '86' in its name somewhere. It might be better to use `^[ix].*86`, or to maintain specfic cases to avoid mismatch.

Also forgive me for a small nit - if we do keep this regexp, the `^` is redundant :)